Software/vortex_ws/src/vision/README.md

```diff
@@ -1,15 +1,15 @@
 # Vision
 
 ## Description
-Vision is a ros2 package which responsible for mission recognition and object detection. 
-* The vision module passes the data to the mission planning module.
-* Data contain logs from the deep learning model and image analysis model.
-* The logs from the deep learning models contain the detected object class ,and it's coordinates
+Vision is a ros2 package of the AUV vision module, which responsible for mission recognition and object detection. 
+
+* It passes the data which contain the logs from the deep learning model and image analysis model to the AUV mission planning module.
+
+* Logs from the deep learning models contain the detected object class ,and it's coordinates
 ,and score of detection. 
-* The logs of the image analysis model contain image manifest, small object coordinates 
+* Logs of the image analysis model contain image manifest, small object coordinates 
 ,and parameter values which may vary according to each task.
-* Both the deep learning model ,and the image analysis model require a noise-free image.
-* So, The input frame (from the camera (Sensor Fusion module)) is passed to a image enhancement node
+* Both the deep learning model and the image analysis model require a noise-free image ,so The input frame (from the camera (Sensor Fusion module)) is passed to a image enhancement node
 to manipulates the image to fit the tasks.
```

Comment:
Hard for me to understand what is the meaning of this, please write it in more descriptive way